packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -179,19 +255,16 @@ export function cloudflare<T extends Record<string, WorkerOptions>>(
 				});
 			};
 		},
-		configurePreviewServer(vitePreviewServer) {
+		async configurePreviewServer(vitePreviewServer) {
 			const miniflare = new Miniflare(
-				getPreviewMiniflareOptions(normalizedPluginConfig, viteConfig),
+				getPreviewMiniflareOptions(resolvedPluginConfig, vitePreviewServer),
 			);
 
-			const middleware = createMiddleware(
-				({ request }) => {
-					return miniflare.dispatchFetch(toMiniflareRequest(request), {
-						redirect: 'manual',
-					}) as any;
-				},
-				{ alwaysCallNext: false },
```

Comment:
Out of curiosity - why is this no longer needed?

Reply:
I think this is only relevant when you have other plugins that add middleware so I thought it's best to remove it for now and revisit when we look at framework integrations.